source/GlassView.Export/Extensions.cs

```diff
@@ -0,0 +1,63 @@
+using System.Text.Json;
+using Atmoos.GlassView.Core;
+
+namespace Atmoos.GlassView.Export;
+
+public static class Extensions
+{
+    private static readonly EnumerationOptions findLeafOptions = new() {
+        RecurseSubdirectories = false,
+        IgnoreInaccessible = true,
+        MatchType = MatchType.Simple,
+        MatchCasing = MatchCasing.CaseSensitive,
+        AttributesToSkip = FileAttributes.Hidden | FileAttributes.System
+    };
+
+    internal static String Serialize<T>(this T value) => Serialize(value, Options(new JsonSerializerOptions()));
+    internal static String Serialize<T>(this T value, JsonSerializerOptions options) => JsonSerializer.Serialize(value, options);
+
+    private static JsonSerializerOptions Options(JsonSerializerOptions options)
+    {
+        options.WriteIndented = true;
+        return options.EnableGlassView();
+    }
+
+    /// <summary>
+    /// Recursively looks upward toward parent directories for the leaf directory <paramref name="leafDirectoryName"/>
+    /// in the current directory structure staring at the current working directory.
+    /// </summary>
+    /// <param name="leafDirectoryName">The leaf directories name.</param>
+    /// <returns>If found, returns that directory.
+    /// When not found, returns the leaf directory as subdirectory of the current directory.</returns>
+    internal static DirectoryInfo FindLeaf(String leafDirectoryName)
```

Comment:
I feel like this extension method would be an  excellent candidate for the Nuget Package from Atmosphere which wraps File & Directory operations.

Reply:
Will move it in due course. However, let's not wait for Atmoos.Sphere.
Created issue [#17](https://github.com/atmoos/Sphere/issues/17) in that repo.